src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -414,13 +417,50 @@ export const disabilityOption = ({ diagnosticCode, name, ratingPercentage }) =>
   );
 };
 
+export const PRESTART_STATUSES = {
+  none: 'none',
+  expired: 'expired',
+  created: 'created',
+  retrieved: 'retrieved',
+  renewed: 'renewed',
+  ...commonPrestartStatuses
+};
+
+export const prestartPendingStatuses = new Set([PRESTART_STATUSES.none, PRESTART_STATUSES.expired, PRESTART_STATUSES.pending]);
+
+const successHeading = {
+  created: 'We’ve verified your Intent to File',
+  retrieved: 'We’ve retrieved your existing Intent to File',
+  renewed: 'We’ve verified your Intent to File'
+};
+
+// TODO: add expired data to redux
+function successMessage(status, data, expiredData) {
+  const messages = {
+    created: (expirationDateString) => `Thank you for submitting your Intent to File for disability compensation. Your Intent to File will expire on ${expirationDateString}.`,
+    retrieved: (expirationDateString) => `Our records show that you’ve already submitted an Intent to File for disability compensation. Your Intent to File will expire on ${expirationDateString}.`,
+    renewed: (expirationDateString, expiredDateString) => `Your existing Intent to File expired on ${expiredDateString}, so we’ve created a new one for you. This new Intent to File request will expire on ${expirationDateString}.`
+  };
+  return messages[status](moment(data).format('M/D/YYYY'), moment(expiredData).format('M/D/YYYY'));
+}
```

Comment:
@peggygannon could you please share your thoughts on these success alert messages?